doc/source/management/changed_page_tracking.rst

```diff
@@ -4,7 +4,13 @@
 XtraDB changed page tracking
 =============================
 
-|XtraDB| now tracks the pages that have changes written to them according to the redo log. This information is written out in special changed page bitmap files.  This information can be used to speed up incremental backups using `Percona XtraBackup <http://www.percona.com/doc/percona-xtrabackup/>`_ by removing the need to scan whole data files to find the changed pages. Changed page tracking is done by a new |XtraDB| worker thread that reads and parses log records between checkpoints. The tracking is controlled by a new read-only server variable :variable:`innodb_track_changed_pages`.
+.. Important:: 
+
+        Starting with Percona Server for MySQL 8.0.26-17, the changed page tracking functionality is deprecated and may be removed in future versions.
+
+        We recommend using `MySQL InnoDB Clone and page tracking <https://dev.mysql.com/blog-archive/innodb-clone-and-page-tracking>`_ .
```

Comment:
We recommend using the MySQL page tracking function. For more information, see

Reply:
Updated